requirements-docs.txt

```diff
@@ -3,5 +3,5 @@ ipython
 ipywidgets
 matplotlib
 numpydoc
-sphinx==1.6.7
+sphinx>=1.8.1,!=2.0.0
```

Comment:
Why do we need to exclude v2.0.0?

Reply:
It picked a much newer version 4.0.1 anyway, so I don't think it's a blocking question.

Reply:
> Why do we need to exclude v2.0.0?

That comes from [`matplotlib` docs requirements](https://github.com/matplotlib/matplotlib/blob/master/requirements/doc/doc-requirements.txt).

Reply:
Cool! Posting the permalink here for the future us, just in case: https://github.com/matplotlib/matplotlib/blob/4723f8a42c8190f5cb287aca8fdf80ec6c37b102/requirements/doc/doc-requirements.txt#L10.

Reply:
I'm using Sphinx 3.? or higher on my side.  If the server builds the docs, I'm OK with closing #986.